addon/templates/components/cfb-loading-dots.hbs

```diff
@@ -0,0 +1 @@
+{{#if hasBlock}}{{yield}}{{else}}{{t 'caluma.form-builder.global.loading'}}{{/if}}{{#each (range 0 count) as |i|}}<span style="--cfb-loading-dots-delay: {{i}};">.</span>{{/each}}
```

Comment:
Might be easier to read if this wasn't a one-liner

Reply:
Yeah I totally agree. I only had to do this because of the whitespaces which I don't want there.. Maybe there is another way?

Reply:
Isn't there the `~` syntax for whitespace control?

Reply:
Yeah, however that does not solve the whitespace between `{{/if}}` and `{{each}}`

Reply:
Or does it? Not sure anymore after reading the handlebars docs :joy:

Reply:
"Template whitespace may be omitted from either side of any mustache statement by adding a ~ character by the braces."
Sound like `{{/if~}}{{~#each ...}}` should work.